opm/input/eclipse/EclipseState/Grid/FieldProps.hpp

```diff
@@ -131,6 +131,7 @@ namespace ALIAS {
 
 namespace GRID {
 static const std::unordered_map<std::string, keyword_info<double>> double_keywords = {{"DISPERC",keyword_info<double>{}.unit_string("Length")},
+                                                                                      {"MINPVV",  keyword_info<double>{}.unit_string("ReservoirVolume")},
```

Comment:
I would hope that the unit_string is not needed and is implied by the json definition of MINPVV